Shell/SimplifyFalseTrue.cpp

```diff
@@ -371,12 +371,12 @@ TermList SimplifyFalseTrue::simplify(TermList ts)
         #define THEN 0u
         #define ELSE 1u
 
-        TermList branches[2];
-        bool isTrue[2];
-        bool isFalse[2];
-        for (BRANCH branch : {THEN, ELSE }) {
-          branches[branch] = simplify(*term->nthArgument(branch));
-        }
+        TermList branches[2] = {
+          simplify(*term->nthArgument(THEN)),
+          simplify(*term->nthArgument(ELSE)),
+        };
+        bool isTrue[2] {};
+        bool isFalse[2] {};
 
```

Comment:
Same change as in NewCNF.

Reply:
Is 

`bool isTrue[2] {};`

the same as 

`bool isTrue[2] {false, false};`

?

Reply:
I think so. https://en.cppreference.com/w/cpp/language/zero_initialization says "If T is array type, each element is zero-initialized."

Reply:
I need to get used to see-ing {} as an initializer. Not sure if we should insist on the more explicit bool isTrue[2] {false, false};

Reply:
I agree it's a bit of an odd syntax choice! I'm OK with whichever - `T t{};` where `T` is non-POD is quite helpful though to avoid `T t();` (function declaration) or `T t = T();` (unnecessary copy).